### PR TITLE
Some cleaning

### PR DESCRIPTION
**Pull request by valeriabarra**
This is a PR of small things that I've noticed here and there.

### CODE REVIEW

File: examples/petsc/bpsraw.c [hunk 1]
```diff
@@ -809,6 +809,12 @@ int main(int argc, char **argv) {
     ierr = KSPGetIterationNumber(ksp, &its); CHKERRQ(ierr);
     ierr = KSPGetResidualNorm(ksp, &rnorm); CHKERRQ(ierr);
     if (!test_mode || reason < 0 || rnorm > 1e-8) {
+      CeedInt gsize;
```
Comment (jeremylt) [inline examples/petsc/bpsraw.c]: I think we already have gsize floating around and defined. Should we remove these extra two lines in all 4 examples?

**Comment by jeremylt** [comment]
Would you like to cherry pick https://github.com/CEED/libCEED/pull/399/commits/36437134ccd3f2e6dabf0771a357ccd4721cfd16 into this branch as well? It fits thematically.

File: examples/petsc/bpsraw.c (hunk 1 above)
Comment (valeriabarra) [reply examples/petsc/bpsraw.c]: Done, but it was only in this one example that it was declared in the block scope

**Comment by valeriabarra** [comment]
> Would you like to cherry pick [3643713](https://github.com/CEED/libCEED/commit/36437134ccd3f2e6dabf0771a357ccd4721cfd16) into this branch as well? It fits thematically.

Done. If fits. Thanks! I had forgotten to update that after the naming convention update

**Pull request merged**